src/langsmith/assistants.mdx

```diff
@@ -3,36 +3,179 @@ title: Assistants
 sidebarTitle: Overview
 ---
 
-**Assistants** allow you to manage configurations (like prompts, LLM selection, tools) separately from your graph's core logic, enabling rapid changes that don't alter the graph architecture. It is a way to create multiple specialized versions of the same graph architecture, each optimized for different use cases through configuration variations rather than structural changes.
+_Assistants_ allow you to manage configurations (like prompts, LLM selection, tools) separately from your graph's core logic, enabling rapid changes that don't alter the graph architecture. It is a way to create multiple specialized versions of the same graph architecture, each optimized for different [use cases](#when-to-use-assistants) through configuration variations rather than structural changes.
 
 For example, imagine a general-purpose writing agent built on a common graph architecture. While the structure remains the same, different writing styles—such as blog posts and tweets—require tailored configurations to optimize performance. To support these variations, you can create multiple assistants (e.g., one for blogs and another for tweets) that share the underlying graph but differ in model selection and system prompt.
 
 ![assistant versions](/langsmith/images/assistants.png)
 
-The LangGraph API provides several endpoints for creating and managing assistants and their versions. See the [API reference](https://langchain-ai.github.io/langgraph/cloud/reference/api/api_ref/#tag/assistants) for more details.
+The LangGraph API provides several endpoints for creating and managing assistants and their versions. See the @[API reference][AssistantsAPI] for more details.
 
 <Info>
-Assistants are a [LangSmith](/langsmith/home) concept. They are not available in the open source LangGraph library.
+Assistants are a [LangSmith Deployment](/langsmith/deployments) concept. They are not available in the open source LangGraph library.
 </Info>
 
-## Configuration
+## When to use assistants
+
+Assistants are ideal when you need to deploy the same graph logic with different configurations. Common use cases include:
```

Comment:
This is a great list of use cases. Each are worthy of their own documentation page (with example code).